conceptpower/classes.py

```diff
@@ -236,8 +236,8 @@ def create(self, user, password, label, pos, conceptlist, description,
             "description": description,
             "type": concepttype,
             "synonymids": synonym_ids,
-            "equals": equal_uris,
-            "similar": similar_uris
+            "equal_to": ",".join(str(equal_uris_element) for equal_uris_element in equal_uris),
+            "similar": ",".join(str(similar_uris_element) for similar_uris_element in similar_uris)
```

Comment:
aren't the elements in ```equal_uris``` strings?

Reply:
I have removed the string typecast in both <b>equal_to</b> and <b>similar</b> fields